tests/UI/campaigns/regression/checkout/100PercentDiscount_FO.js

```diff
@@ -0,0 +1,417 @@
+      );
+
+      // Check that we got to order confirmation (probably not necessary)
+      const cardTitle = await orderConfirmationPage.getOrderConfirmationCardTitle(
+        page,
+      );
+
+      await expect(cardTitle).to.contains(
+        orderConfirmationPage.orderConfirmationCardTitle,
+      );
+    });
+  });
+
+  /**
+   * Cleanup
+   * Remove the discount code and the generated order
+   * Change the terms and conditions setting back to enabled
+   */
+  describe('CLEANUP', async () => {
+    it('should go back to BO', async function () {
+      await testContext.addContextItem(
+        this,
+        'testIdentifier',
+        'BackToBOForCleanup',
+        baseContext,
+      );
+
+      page = await checkoutPage.closePage(browserContext, page, 0);
+
+      const pageTitle = await orderSettingsPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(orderSettingsPage.pageTitle);
+    });
+
+    describe('Delete created cart rules', async () => {
+      it('should go to cart rules page', async function () {
+        await testContext.addContextItem(
+          this,
+          'testIdentifier',
+          'goToCartRulesPageToDelete',
+          baseContext,
+        );
+
+        await cartRulesPage.goToSubMenu(
+          page,
+          cartRulesPage.catalogParentLink,
+          cartRulesPage.discountsLink,
+        );
+
+        const pageTitle = await cartRulesPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(cartRulesPage.pageTitle);
+      });
+
+      it('should delete our 100% cart rules', async function () {
+        await testContext.addContextItem(
+          this,
+          'testIdentifier',
+          'deleteCartRules',
+          baseContext,
+        );
+
+        const validationMessage = await cartRulesPage.deleteCartRule(page);
+        await expect(validationMessage).to.contains(
+          cartRulesPage.successfulDeleteMessage,
+        );
+      });
+    });
+
+    describe('Change terms and conditions setting back to yes', async () => {
+      it("should go to 'Shop Parameters > Order Settings' page", async function () {
+        await testContext.addContextItem(
+          this,
+          'testIdentifier',
+          'goToOrderSettingsPage',
+          baseContext,
+        );
+
+        await dashboardPage.goToSubMenu(
+          page,
+          dashboardPage.shopParametersParentLink,
+          dashboardPage.orderSettingsLink,
+        );
+
+        const pageTitle = await orderSettingsPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(orderSettingsPage.pageTitle);
+      });
+      it('Should change the terms and conditions back to enabled', async () => {
+        const result = await orderSettingsPage.setTermsOfService(
+          page,
+          true,
+          'Terms and conditions of use',
+        );
+
+        await expect(result).to.contains(
+          orderSettingsPage.successfulUpdateMessage,
+        );
+      });
```

Comment:
```suggestion
      it('should change the terms and conditions back to enabled', async function () {
        await testContext.addContextItem(
          this,
          'testIdentifier',
          'resetTermsAndConditionsValue',
          baseContext,
        );

        const result = await orderSettingsPage.setTermsOfService(
          page,
          true,
          'Terms and conditions of use',
        );

        await expect(result).to.contains(
          orderSettingsPage.successfulUpdateMessage,
        );
      });
```

Step identifier was missing here